test/unit/ui/map.test.js

```diff
@@ -77,6 +77,16 @@ test('Map', (t) => {
         t.end();
     });
 
+    t.test('initial bounds in constructor options', (t) => {
+        const bounds = [[-133, 16], [-68, 50]];
+        const map = createMap({bounds});
+
+        t.deepEqual(fixedLngLat(map.getCenter(), 4), { lng: -100.5, lat: 34.7171 });
+        t.equal(fixedNum(map.getZoom(), 3), 2.469);
+
+        t.end();
+    });
+
```

Comment:
> But the same steps reproduced in test somehow outcomes with different zoom level

The zoom level depends on the size of the map container.

Reply:
I took https://github.com/mapbox/mapbox-gl-js/blob/master/test/unit/ui/camera.test.js#L1355 as an example. And it works fine with the zoom level.